stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/composer/MessageComposerController.kt

```diff
@@ -43,6 +60,32 @@ public class MessageComposerController(
      */
     public val selectedAttachments: MutableStateFlow<List<Attachment>> = MutableStateFlow(emptyList())
 
+    /**
+     * Represents the list of validation errors for the current text input and the currently selected attachments.
+     */
+    public val validationErrors: MutableStateFlow<List<ValidationError>> = MutableStateFlow(emptyList())
+
+    /**
+     * Represents the maximum allowed message length in the message input.
+     */
+    private var maxMessageLength: Int = DEFAULT_MAX_MESSAGE_LENGTH
+
+    /**
+     * Sets up the data loading operations such as observing the maximum allowed message length.
+     */
+    init {
+        scope.launch {
+            val result = chatDomain.watchChannel(channelId, 0).await()
+
+            if (result.isSuccess) {
+                val channelController = result.data()
+                channelController.channelConfig.collect {
+                    maxMessageLength = it.maxMessageLength
+                }
```

Comment:
This is a black box, it can emit couple of times the default value, then a value from the server. Need to discuss this.

Reply:
I think we can observe it and all, but we should definitely follow up with BE to see what the issue here is for inconsistency